app/views/interstitials/_scl_provision_user.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1 class="govuk-heading-xl">
+  Get laptops and internet access
+</h1>
+
+<p class="govuk-body">
+  Laptops are now available for care leavers aged 16 to 25 who <%= @current_user.effective_responsible_body.name %> is responsible for.
+</p>
+
+<p class="govuk-body">
+  Use this service to:
+</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>check you’ve been allocated the correct number of laptops</li>
+  <li>place orders for laptops</li>
+  <li>request internet access for eligible pupils</li>
```

Comment:
Please change 'pupils' to 'care leavers', thanks

Reply:
done